.gitlab-ci.yml

```diff
@@ -732,7 +732,7 @@ zombienet-tests-malus-dispute-valid:
     - echo "${PARACHAINS_IMAGE_NAME} ${PARACHAINS_IMAGE_TAG}"
     - echo "${MALUS_IMAGE_NAME} ${MALUS_IMAGE_TAG}"
     - echo "${GH_DIR}"
-    - export DEBUG=zombie*
+    - export DEBUG=zombie*,zomobie::port-forwarder
```

Comment:
@pepoviola is this a type or intentional?

Reply:
Hi @drahnr, is intentional but is missing the `-` to don't print debugs from the `port-forwarder`.
Thanks!

Reply:
I think @drahnr meant a typo in `zomobie -> zombie` :)

Reply:
ahh that true, I will fix asap. Thanks!!